modules/tide_content_collection/src/Plugin/Field/FieldWidget/ContentCollectionConfigurationWidget.php

```diff
@@ -847,6 +848,10 @@ protected function buildContentTabAdvancedFilters(FieldItemListInterface $items,
             ];
             $field_filter['#title'] = $this->t('Add @label filters', ['@label' => strtolower($field_label)]);
             $element['tabs']['content']['advanced_filters'][$field_id . '_wrapper'][$field_id] = $field_filter;
+            if ($field_id === 'field_node_site') {
```

Comment:
tide_api and tide_content_collection module doesn't have dependency to `tide_site`.

Reply:
Added check for tide_site exists.